app/main.py

```diff
@@ -1,3 +1,24 @@
-def shop_trip():
-    # write your code here
-    pass
+import json
+from pathlib import Path
+
+from app.customer import Customers
+from app.shops import Shops
+
+
+def shop_trip() -> None:
+    path_ = Path(__file__).parent.parent
+    path_new = path_.joinpath("app").joinpath("config.json")
+    with open(path_new, "r") as data_file:
+        config_file = json.load(data_file)
+        fuel_price = config_file["FUEL_PRICE"]
+
+        for shop in config_file["shops"]:
+            market = Shops(shop)
+        for customer in config_file["customers"]:
+            person = Customers(customer)
+
+            Customers.choice_of_options(person, market, fuel_price)
```

Comment:
You don't need to pass market inside this function because you already iterate through list of shops inside it

Reply:
Call it in such a way - person.choice_of_options - because it is an instance method

Reply:
And the method name itself is not very descriptive